templates/fine-tune-llm_v2/README.md

```diff
@@ -0,0 +1,85 @@
+# Fine-tuning Llama-2/Mistral models with Anyscale
+
+The guide below walks you through the steps required for fine-tuning of LLM models. This template provides an easy to configure solution for ML Platform teams, Infrastructure engineers, and Developers to fine-tune LLMs.
+
+### Supported base models
+
+- mistralai/Mistral-7B-Instruct-v0.1
+- mistralai/Mixtral-8x7b
+- meta-llama/Llama-2-7b-hf
+- meta-llama/Llama-2-7b-chat-hf
+- meta-llama/Llama-2-13b-hf
+- meta-llama/Llama-2-13b-chat-hf
+- meta-llama/Llama-2-70b-hf
+- meta-llama/Llama-2-70b-chat-hf
+
+## Step 1 - Launch a fine-tuning job
+
+We have provided different example configurations under the `training_configs`
+directory for different base models and accelerator types. You can use these as a
+starting point for your own fine-tuning jobs.
+
+[Optional] you can get a WandB API key from [WandB](https://wandb.ai/authorize) to track the finetuning process.
+
+Next, you can launch a fine-tuning job where the WandB API key is passed as an environment variable.
+
+
+
+From the VSCode terminal (press [**Ctrl + `**] in VSCode), run the command to trigger a fine-tuning job. Generally, a fine-tuning job will take a few hours.
```

Comment:
Can you add some screenshots of the expected stdout / how to monitor the job in the Ray dashboard?